README.md

```diff
@@ -26,10 +26,22 @@ If you are migrating a service to DBT PaaS, [GOV.UK PaaS to DBT PaaS Migration](
 
 ### Testing
 
+#### Automated testing
+
 Run `poetry run pytest` in the root directory to run all tests.
 
 Or, run `poetry run tox` in the root directory to run all tests for multiple Python versions. See the [`tox` configuration file](tox.ini).
 
+#### Manual testing
+
+You may want to test any CLI changes locally. 
+
+Run `poetry build` to build your package resulting in a package file (e.g. `dbt_copilot_tools-0.1.40.tar.gz`) in a `dist` folder. You may need to bump up the package version before doing so.
+
+Copy the package file(s) to a repository where you would like to test your changes, and make sure you are in a virtual environment.
```

Comment:
This worked for me BTW: pip install ../copilot-tools/dist/dbt_copilot_tools-0.1.46.tar.gz

So no need to move the files anywhere.

Reply:
Also, "repository" is a bit of an overloaded term (git repo? pypi repo?), so "directory" may be better. But probably just don't need to copy it anywhere.

Reply:
It's a lot easier to copy and paste the file, because we don't know what everyone's folder structure is like. I'll add a comment that is unnecessary though.

I'll also change "repository" to "directory" which makes a lot more sense!

Reply:
Resolved in https://github.com/uktrade/copilot-tools/pull/171/commits/02ac3957c216ed8f024cbf9c8a12dbfc5b33394d.